presto-hive/src/main/java/com/facebook/presto/hive/HiveTableProperties.java

```diff
@@ -88,6 +92,27 @@ public HiveTableProperties(TypeManager typeManager, HiveClientConfig config)
                                 .map(name -> ((String) name).toLowerCase(ENGLISH))
                                 .collect(Collectors.toList())),
                         value -> value),
+                // For sorted_by, we use (+/-)col to represent sorting columns, where + means ascending and - means descending.
+                new PropertyMetadata<>(
+                        SORTED_BY_PROPERTY,
+                        "sorting columns",
+                        typeManager.getType(parseTypeSignature("array(varchar)")),
+                        List.class,
+                        ImmutableList.of(),
+                        false,
+                        value -> ((Collection<?>) value).stream()
+                                .map(name -> {
+                                    checkState(
+                                            ((String) name).charAt(0) == '+' || ((String) name).charAt(0) == '-',
+                                            "Expect +/- to specify ASC/DESC for sorting columns");
```

Comment:
`s/Expect to specify/Expected to denote` ?
Why not use `<col> ASC` here? Semantics would be more obvious.